research-app/src/App.vue

```diff
@@ -759,6 +756,7 @@ class TableLayerMessageHandler {
     if (msg.settings.length !== msg.values.length) return;
 
     const layer = this.owner.spreadSheetLayerById(msg.id);
+    console.log(layer);
```

Comment:
I'm guessing this is some debugging code from unrelated work? It shouldn't be included here. (I find `git add -p` to be helpful for this kind of stuff.)